client/src/components/App/App.tsx

```diff
@@ -38,29 +38,75 @@ const App: React.FC = (): JSX.Element => {
     const [isUserUpdated, setIsUserUpdated] = useState<boolean>(true);
 
     const setCurrentAndGroupUsers = (userId: string, userName: string, userToken: string) => {
+        logger.info({
+            service: Service.CLIENT,
+            severity: Severity.LOW,
+            workflow: 'Getting group users',
```

Comment:
commented on other PRS, but is there a reason to log so many info logs on client side? 
we already have the same logs on the server, and if the request fails we can still log the error? 🤔 🤷‍♀️

Reply:
So we'd have the whole flow, from client to server monitored

Reply:
I just think its redundant and makes the code and the logs harder to read

Reply:
but if its what was decided then sure

Reply:
I think this is what is has been decided, but we can bring it up again